packages/mobile/ios/Podfile

```diff
@@ -61,8 +61,13 @@ target 'celo' do
   pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
   pod 'CeloBlockchain', :path => '../../../node_modules/@celo/client/CeloBlockchain.podspec'
   pod 'RNSecureKeyStore', :path => '../../../node_modules/react-native-secure-key-store/ios'
+  pod 'react-native-safe-area-context', :path => '../../../node_modules/react-native-safe-area-context'
```

Comment:
just curious, does the RN link command work for iOS or is that still broken and we do these by hand?

Reply:
Still broken, I think it's because it doesn't like our monorepo. I think autolinking should avoid this problem.